src/Card/Card.scss

```diff
@@ -133,6 +153,10 @@
       width: 100%;
       margin-top: $card-footer-actions-gap;
     }
+
+    .react-loading-skeleton {
+      min-height: 9rem;
```

Comment:
Should we follow a similar approach to the height for the skeleton as in the `CardSection` component versus relying on CSS here? 

```jsx
<Skeleton containerClassName="pgn__card-section-loader" height={100} />
```

Reply:
Done